plugins/search-react/src/components/SearchResultGroup/SearchResultGroup.tsx

```diff
@@ -0,0 +1,489 @@
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, {
+  ChangeEvent,
+  PropsWithChildren,
+  ReactNode,
+  useCallback,
+  useState,
+} from 'react';
+import useAsync from 'react-use/lib/useAsync';
+import qs from 'qs';
+
+import {
+  makeStyles,
+  Theme,
+  List,
+  ListSubheader,
+  Typography,
+  Chip,
+  InputBase,
+  MenuItem,
+  Select,
+  Menu,
+  TypographyProps,
+  ListItem,
+  ListItemText,
+} from '@material-ui/core';
+import AddIcon from '@material-ui/icons/Add';
+import ArrowRightIcon from '@material-ui/icons/ArrowForwardIos';
+
+import { JsonValue } from '@backstage/types';
+import { Link, LinkProps } from '@backstage/core-components';
+import { AnalyticsContext, useApi } from '@backstage/core-plugin-api';
+import { SearchQuery, SearchResult } from '@backstage/plugin-search-common';
+
+import { DefaultResultListItem } from '../DefaultResultListItem';
+import { searchApiRef } from '../../api';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    background: theme.palette.background.paper,
+    '&:not(:first-child)': {
+      marginTop: theme.spacing(3),
+    },
+  },
+  listSubheader: {
+    display: 'flex',
+    alignItems: 'center',
+  },
+  listSubheaderName: {
+    marginLeft: theme.spacing(1),
+    textTransform: 'uppercase',
+  },
+  listSubheaderChip: {
+    color: theme.palette.text.secondary,
+    margin: theme.spacing(0, 0, 0, 1.5),
+  },
+  listSubheaderFilter: {
+    display: 'flex',
+    color: theme.palette.text.secondary,
+    margin: theme.spacing(0, 0, 0, 1.5),
+  },
+  listSubheaderLink: {
+    marginLeft: 'auto',
+    display: 'flex',
+    alignItems: 'center',
+  },
+  listSubheaderLinkIcon: {
+    fontSize: 'inherit',
+    marginLeft: theme.spacing(0.5),
+  },
+}));
+
+/**
+ * Props for {@link SearchResultGroupFilterFieldLayout}
+ * @public
+ */
+export type SearchResultGroupFilterFieldLayoutProps = PropsWithChildren<{
+  label: string;
+  value?: JsonValue;
+  onDelete: () => void;
+}>;
+
+/**
+ * Default layout for a search group filter field.
+ * @param props - see {@link SearchResultGroupFilterFieldLayoutProps}.
+ * @public
+ */
+export const SearchResultGroupFilterFieldLayout = (
+  props: SearchResultGroupFilterFieldLayoutProps,
+) => {
+  const classes = useStyles();
+  const { label, children, ...rest } = props;
+
+  return (
+    <Chip
+      {...rest}
+      className={classes.listSubheaderFilter}
+      variant="outlined"
+      label={
+        <>
+          {label}: {children}
+        </>
+      }
+    />
+  );
+};
+
+const NullIcon = () => null;
```

Comment:
should we have a default icon instead?

Reply:
Do you mean a default icon for the select chip filter?
I removed the arrow down icon to have the same appearance as the text filter for filters having the same appearance when they are not in editing mode...
Do you want me to add it back?